### DISCUSSION TITLE
Interactive Image Viewer

### DISCUSSION THREAD

**Discussion by adamviola** [Textualize/textual#1519] [Show and tell]
I've been working on an interactive image viewer using Textual under the name [`textual-imageview`](https://github.com/adamviola/textual-imageview)

![cat](https://user-images.githubusercontent.com/43352940/210931390-ad1f47fc-2340-435e-8851-234b5fa96d0f.gif)

The app is built on a Rich renderable that renders the image (each character as two pixels), and a Textual widget that uses mouse dragging and scrolling to manipulate the scale and translation.

This is my first go at Textual widget/app. Let me know what you think!

**Comment by davep** [comment]
That's awesome. Going to have a play with that!
